projects/packages/search/src/dashboard/components/record-meter/notice-box.jsx

```diff
@@ -14,7 +14,7 @@ import NoticeAction from 'components/notice/notice-action.jsx';
 const CLOSE_TO_LIMIT_PERCENT = 0.8;
 const DISMISSED_NOTICES = 'jetpack-search-dismissed-notices';
 
-const getNotices = ( planRecordLimit = null ) => {
+const getNotices = ( tierMaximumRecords = null ) => {
```

Comment:
We were using `planRecordLimit` and `tierMaximumRecords` in different places and they mean the same thing. Consolidated on `tierMaximumRecords`.